lib/device_api/android/device.rb

```diff
@@ -54,14 +59,14 @@ def status
       end
       
       def connect
-        ADB.connect(@ip_address, @port)
+        ADB.connect(ip_address, port)
```

Comment:
Its the same pattern followed in device_api-android. However, as discussed we should come up with some standard rules.